tests/integration/gateway/anycast/vlan.yml

```diff
@@ -0,0 +1,52 @@
+#
+# Anycast gateway on a physical interface
+#
+
+message: |
+  This test case replaces a multi-access LAN with a VLAN -- the switches have anycast
+  IP address configured on the shared LAN. Two LANs are used in the topology to explore
+  the impact of shared MAC addresses on multiple LANs.
+
+  Static routes on hosts point to the shared anycast IP address. The switches
+  are running OSPF with a third switch to enable network-wide connectivity.
+
+  All hosts should be able to ping the loopback addresses of all switches.
+
+gateway.id: 1
+
+groups:
+  switches:
+    module: [ gateway,ospf,vlan ]
+    members: [ s1,s2,s3 ]
+  hosts:
+    members: [ h1, h2, h3, h4 ]
+    device: linux
+
+vlans:
+  red:
+    gateway: True
+  blue:
+    gateway: True
+
+nodes: [ s1,s2,s3,h1,h2,h3,h4 ]
+
+links:
+- s1:
+  h1:
+  vlan.access: red
+- s2:
+  h2:
+  vlan.access: red
+- s1:
+  h3:
+  vlan.access: red
```

Comment:
At least 1 of the hosts should be on 'blue', else nobody can reach the VRRP gateway there

Reply:
Ah, the beautiful copy/paste artifacts. Of course you're absolutely correct, fixed.